### PR TITLE
Partner view tests

### PR DESCRIPTION
**Pull request by kierangillen**
Adds tests for new partner view components

#skip_new_tests

### CODE REVIEW

**Comment by ds300** [comment]
:wave: we're looking into using the new relay test utils more and the next step is to try them on a pagination container. Might make sense to do that for this page, if you're down to pair on it some time?

File: src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            }
+          }
+        }
+      `,
+      mockData: {
+        partner,
+      },
+    })
+
+  it("It renders the artworks", async () => {
```
Comment (ashfurrow) [inline src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx]:
```suggestion
  it("renders the artworks", async () => {
```

Note that we'll need to re-do the snapshots after this change. There are also a few others we'll want to change.

File: src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const wrapper = await getWrapper(PartnerArtworkFixture as any)
+    const grid = wrapper.find(GenericGrid)
+    expect(grid.props().artworks.length).toBe(10)
+    expect(grid.length).toBe(1)
+    expect(wrapper.html()).toMatchSnapshot()
```
Comment (ashfurrow) [inline src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx]: Should this be `.html()` or `.toJSON()`?

Comment (kierangillen) [reply src/lib/Scenes/Partner/Components/__tests__/PartnerArtwork-tests.tsx]: We're using `html()` in other instances, which if there are diffs should be easier to track down the code changes. Both presumably would work though to capture any changes.

**Comment by artsyit** [comment]
:rocket: PR was released in v1.18.29 :rocket:

**Pull request merged**